src/shared/systems/collision.sibilant

```diff
@@ -35,8 +35,8 @@
 
   ;; (gett x this.pos.x)
   ;; (gett y this.pos.y)
-  (gett height (Math.round (* 2 this.scale)) )
-  (gett width (Math.round (* 2 this.scale)))
+  (gett height (* 2 this.scale) )
```

Comment:
We are still having some random issues with collision.. I don't think it is an issue with the implementation of this system though, I think it's an interaction that hapens in the event handler.

Reply:
When I've changed the order of the systems updates I think this stopped happening all together, but it got very laggy because there were a lot more movement updates happening, and the screen became quite noisy with wobbling rocks.

Reply:
Add a task for this.